plugins/ruby-foreman-remote-execution/debian/install

```diff
@@ -1,4 +1,4 @@
 foreman_remote_execution.rb           usr/share/foreman/bundler.d
 cache                                 usr/share/foreman/vendor
-foreman_remote_execution              var/lib/foreman/public/assets
+assets                                var/lib/foreman/public
```

Comment:
foreman_remote_execution assets aren't namespaced, so we have to do something like this.

Not sure about the implications of this though - will non-plugin assets end up in the DEB too?